spec/asyncapi.md

```diff
@@ -537,17 +538,14 @@ Field Pattern | Type | Description
 ```json
 {
   "user/signedup": {
-    "subscribe": {
-      "$ref": "#/components/messages/userSignedUp"
-    }
+    "subscribe": { }
```

Comment:
Why not the following?

```suggestion
    "subscribe": {
      "$ref": "#/components/messages/userSignedUp"
    }
```

Reply:
You mean:
```json
"subscribe": {
    "message": {
        "$ref": "#/components/messages/userSignedUp"
    }
}
```
right?

I dropped the message here and in the other places you commented because I felt that it didn't add much to the understanding in these sections (explaining the channel item and parameter object).

But definitely not a strong opinion on that one. I changed it to your suggestion.

Reply:
Oh yeah, I forgot the message key again 😅 Yeah, I think it's great to add some context. What you did is perfect IMHO 👍